resource_transip_vps.go

```diff
@@ -17,7 +18,7 @@ func resourceVps() *schema.Resource {
 	return &schema.Resource{
 		Create: resourceVpsCreate,
 		Read:   resourceVpsRead,
-		// Update: resourceVpsUpdate,
+		Update: resourceVpsUpdate,
```

Comment:
By allowing updates we give the expectation to the user all (non-computed) attributes on this resource can be updated. This will be reflected in the plan as well. But the implementation below does not take care of all these situations.

We should either:

1. implement Update fully
2. perform the customer lock update at the end of the create step.

I'm ok with either solution, but the second one seems the quickest to get this feature included for now.

Reply:
I've worked on implementing the update fully, but had some trouble adding `tags` to a VPS. I commented it out for now, but for your information it resulted in the following error:
```bash
panic: interface conversion: interface {} is []interface {}, not []string
```